draft-ietf-tls-tls13.md

```diff
@@ -930,14 +933,10 @@ type
 : The higher-level protocol used to process the enclosed fragment.
 
 version
-: The version of the protocol being employed.  This document
-  describes TLS Version 1.3, which uses the version { 3, 4 }.  The
-  version value 3.4 is historical, deriving from the use of {3, 1}
-  for TLS 1.0.  (See {{record-layer-1}}.)  Note that a client that
-  supports multiple versions of TLS may not know what version will
-  be employed before it receives the ServerHello.  See
-  {{backward-compatibility}} for discussion about what record layer
-  version number should be employed for ClientHello.
+: The version of the protocol being employed in the current record.
+  For the initial ClientHello this MUST be { 3, 1 }.
```

Comment:
Perhaps say 
"The client MUST set the version to {3, 1} for the initial ClientHello."